iot_config/src/main.rs

```diff
@@ -61,7 +62,7 @@ impl Daemon {
         poc_metrics::start_metrics(&settings.metrics)?;
 
         // Create database pool
-        let pool = settings.database.connect(10).await?;
+        let pool = settings.database.connect(50).await?;
```

Comment:
Is this solving something? Should we make it configurable?

Reply:
i wouldn't bother in this case; @bbalser is coming through and adjusting all of the database connections to allow for connecting with either postgres password _or_ aws iam role and this should get sorted out then